getting-started.md

```diff
@@ -179,12 +181,17 @@ For files shared among users (references, databases, etc), use /group/ctbrowngrp
 ## Using shared resources
 
 Users in `ctbrowngrp` collectively share resources. 
-Currently, this group has priority access to 1 TB of ram and 96 CPUs.
-These resources are accessed using the big mem partition, `bm*`.
+Currently, this group has priority access to 1 TB of ram and 96 CPUs on
+one machine, and 256 GB RAM and up to 64 CPUs on another two machines.
```

Comment:
suggest change to "Currently, this group has priority access to 1 TB of ram and 96 CPUs on
one big mem machine..."